src/toil/options/runner.py

```diff
@@ -25,14 +25,14 @@ def add_runner_options(
         help="Run the file imports on a worker instead of the leader. This is useful if the leader is not optimized for high network performance. "
         "If set to true, the argument --importWorkersDisk must also be set."
     )
-    import_workers_disk_arguments = ["--importWorkersDisk"]
+    import_workers_threshold_argument = ["--importWorkersThreshold"]
     if cwl:
-        import_workers_disk_arguments.append("--import-workers-disk")
+        import_workers_threshold_argument.append("--import-workers-threshold")
     parser.add_argument(
-        *import_workers_disk_arguments,
-        dest="import_workers_disk",
+        *import_workers_threshold_argument,
+        dest="import_workers_threshold",
         type=lambda x: human2bytes(str(x)),
-        default=None,
-        help="Specify the amount of disk space an import worker will use. If file streaming for input files is not available, "
-        "this should be set to the size of the largest input file. This must be set in conjunction with the argument runImportsOnWorkers."
+        default="1 GiB",
+        help="Specify the file size threshold that determines if the file import will happen in its own job. All files below the threshold "
+        "will go into a batch import job. This should be set in conjunction with the argument --runImportsOnWorkers."
```

Comment:
This makes it sound like there's just one batch import job, when really we batch all the imports into batches of about this size.